### PR TITLE
Vip go standards

### PR DESCRIPTION
**Pull request by a6software**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,4 +14,4 @@ before_install:
 install:
     - php -d memory_limit=3G ./bin/composer.phar install -o
 script:
-    - "./bin/phpcs . --severity=1"
+    - "./bin/phpcs . --severity=8"
```
Comment (bizmate) [inline .travis.yml:17]: i think we are always running severity 1 as it is the most similar level of severity as per VIP bot, have you checked this passes phpcs on the wp-content project?

Comment (a6software) [reply .travis.yml:17]: It's not a fair comparison - the standards on Peake are much stricter than VIP Go, and all I'm trying to do here is get the plugin code to a point where VIP Bot doesn't find build halting fault with this code.

Comment (a6software) [reply .travis.yml:17]: As mentioned in the QA PR, I can't exactly reproduce the exact fails / errors reported by VIP Bot either. Still not figured that one out.

Comment (jenkoian) [reply .travis.yml:17]: Have we tried ensuring the sniffs are up to date?

Comment (jenkoian) [reply .travis.yml:17]: Not sure I understand the reasoning behind making this lower?

Comment (bizmate) [reply .travis.yml:17]: i think the PR is stuck at the moment. I dont want to block it but i think for sure we should have the sniffs consistently applied also on the gdpr plugins. I can only see the phpdocumentor package on this composer.
I think we need 

```
        "automattic/phpcs-neutron-standard": "^1.5",
        "automattic/vipwpcs": "^0.4",
        "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
        "phpcompatibility/phpcompatibility-wp": "^2.0",
        "squizlabs/php_codesniffer": "^3.3",
        "wp-coding-standards/wpcs": "^1.0",
```

Composer update and install them to the latest version and then running phpcs against with the sniffs applied

Comment (a6software) [reply .travis.yml:17]: I worked to meet the standards that came with the plugin, which was my mistake - https://github.com/trewknowledge/GDPR/blob/develop/phpcs.xml.dist I saw VIP and assumed, wrongly that this was what we were using. `3.3.0` meets the standards from that `phpcs.xml` file. 

It caught me off guard when a QA release triggered various warnings / errors. 

When I reviewed the `phpcs.xml` file against Peake, I realised they were different and changed this one to same as Peake.

When I ran PHPCS I expected to see a few warnings / errors, as per the failed QA release. Instead, I saw ~50. 

I made a start on fixing all of these, but some are quite large changes. 

I thought this was then blocking two QA releases, so upped the severity to get this to be a valid build. At severity 8, I could see the three failures reported by VIP Bot, which I fixed to allow this to be merged, to allow the QA releases to go ahead. 

I wasn't / am not intending to leave this at `8`, but wanted a fix in place to remove the block. 

I will try the above suggestion, and also continue to bring this inline with severity `1` anyway, which (should) negate(s) any other problem. 

I am away for two days next week and didn't want to leave this in an unusable state in that time. 

Hope that clarifies it :D Cheers both.

**Review by jenkoian — APPROVED** [review]

**Review by jenkoian — APPROVED** [review]

**Pull request merged**